kubernetes-elasticsearch-cluster/deploy.sh

```diff
@@ -12,33 +12,59 @@ then
 fi
 
 dataset=$1
-java_opts=$(jq --raw-output '.es_java_opts' dataset_config/${dataset}/deploy.json)
+project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
```

Comment:
nit rename file to deploy.sh, to distinguish this file from create-cluster.sh. Also update README, etc.

Reply:
Done.